site/app/controllers/student/SubmissionController.php

```diff
@@ -1667,27 +1667,31 @@ public function checkRefresh() {
     public function showStats() {
         $course_path = $this->core->getConfig()->getCoursePath();
         $gradeable_id = $_REQUEST['gradeable_id'] ?? '';
-        $json_path = $course_path . "/submissions/" . $gradeable_id . "/";
-        $path_reset = $json_path;
-        $users = array();
-        if(!file_exists($json_path)) {
-            return;
-        }
-        $user_id_arr = array_slice(scandir($json_path), 2);
-        $user = $user_id_arr[0];
-        $users[$user] = array();
-        for($i = 0; $i < count($user_id_arr); ++$i) {
-            $files = scandir($json_path . $user_id_arr[$i]);
+        $base_path = $course_path . "/submissions/" . $gradeable_id . "/";
+        if(!is_dir($base_path))
+                return;
+        $users = array();      
+        $user_id_arr = array_slice(scandir($base_path), 2);
```

Comment:
Can we change these 4 lines above to be:
```
$users = array();
$user_id_arr = is_dir($base_path) ? array_slice(scandir($base_path), 2) : [];
```

so that we we'll just fall through to the bottom of the function and just show the empty tabular page instead of a completely blank page?

Reply:
ok!